libs/pageserver_api/src/key.rs

```diff
@@ -30,24 +29,25 @@ pub const KEY_SIZE: usize = 18;
 /// See [`Key::to_i128`] for more information on the encoding.
 pub const METADATA_KEY_SIZE: usize = 16;
 
-/// The key prefix start range for the metadata keys. All keys with the first byte >= 0x80 is a metadata key.
-pub const METADATA_KEY_BEGIN_PREFIX: u8 = 0x80;
+/// The key prefix start range for the metadata keys. All keys with the first byte >= 0x40 is a metadata key.
+pub const METADATA_KEY_BEGIN_PREFIX: u8 = 0x60;
+pub const METADATA_KEY_END_PREFIX: u8 = 0x7F;
```

Comment:
Doesn't this shrink the metadata keyspace to 0x20?

I'm 100% fine with that change though, but I suppose the PR description should be adjusted.

Reply:
Yep, but once we expand the keyspace to full u128, we can have 0x60-0xFF instead of 0x60-0x7F. I'll update the PR description.

Reply:
> we can have 0x60-0xFF instead of 0x60-0x7F.

I think we can just leave it unassigned for now. No need to fully assign everything, needs might change.